src/main/java/us/ihmc/commons/thread/RepeatingTaskThread.java

```diff
@@ -331,14 +328,8 @@ public final void run()
    {
       while (running)
       {
-         synchronized (executionState)
-         {
-            if (executionState.getScheduled() == 0L)
-            {
-               executionState.waitForChange();
-               continue;
```

Comment:
Oops, this continue is important to have

Reply:
reverted